build_tools/benchmarks/benchmark_helper.py

```diff
@@ -141,12 +146,61 @@ def _dump_cmds_handler(e2e_test_artifacts_dir: pathlib.Path,
   print(*lines, sep="\n")
 
 
+def merge_results(benchmark_results_files: List[io.TextIOWrapper]):
+  # Merges the benchmark results from `right` into `left` and returns the updated `left`
+  def _merge_two_resultsets(left, right):
```

Comment:
If the nested function doesn't need to access any outer variables, feel free to define it as unnested variable at top-level.

Reply:
Moved